arangod/StorageEngine/HotBackup.cpp

```diff
@@ -46,7 +47,7 @@ HotBackup::HotBackup(application_features::ApplicationServer& server)
   } else if (EngineSelectorFeature::isRocksDB()) {
     _engine = BACKUP_ENGINE::ROCKSDB;
   } else {
-    _engine = BACKUP_ENGINE::MMFILES;
+    THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_NOT_IMPLEMENTED, "mmfiles storage engine not available");
```

Comment:
Maybe. same message as below `"hot backup not implemented for this storage engine"`

Reply:
:+1: